healthtools/sms/__init__.py

```diff
@@ -52,8 +53,10 @@ def create_sms(result, doc_type):
         return response
 
     # TODO: Figure out singular vs plural
-    response += 'We found ' + str(result['total']) + ' matches:'
+    response += 'We found ' + str(len(result['hits'])) + ' matches:'
```

Comment:
What's the difference between the two?

Reply:
I was thinking the response might be a bit confusing to the user since str(result['total']) returns all doc that matches the query in any form (say >200)
while str(len(result['hits'])) returns the length of all hits returned by elasticsearch once(top 10 hits).
And again, since we are returning just the top three hits to the user, can we reword the message to something like 
`Here are the top 3 matches your query returned`

Reply:
Yes, let's do that. And add "Find the full list at http://health.the-star.co.ke/

Reply:
Alright. I have updated the pull request to effect the change.